src/components/Books.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+function Books() {
+  return (
+    <section>
+      <form>
+        <input type="text" placeholder="Add a book" />
+        <input type="submit" />
+      </form>
+      <button type="button">Remove</button>
+    </section>
+  );
```

Comment:
- [ ] This `Books` component should render a list of books using the `Book` component. Actually, you are rendering a form that is not requested here. Let's go ahead and refactor this to return a list of books.

Here is an example: 
```
<section>
      <Book title='aaa' author='dd'>
      <Book title='abc' author='da'>
</section>
```

- [ ] Notice that you should create a component `Book` for every single book that takes the book data as a prop and renders the title and the author, and has a button for removing the book (the button wouldn't remove the books yet). Let's do it

- [ ] Finally, you need to create a component `CreateNewBook` that renders a form with inputs for the title and the author of the book and has a button to submit (The button wouldn't add the book yet). This will be necessary to create new books.